AdminApi/test/AdminApi.Tests.Integration/Validators/JsonValidators.cs

```diff
@@ -1,28 +1,38 @@
 ﻿using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
-using Newtonsoft.Json.Schema;
+using NJsonSchema;
 using NJsonSchema.NewtonsoftJson.Generation;
 using JsonSchemaGenerator = NJsonSchema.Generation.JsonSchemaGenerator;
 
 namespace Backbone.AdminApi.Tests.Integration.Validators;
 
 public class JsonValidators
```

Comment:
I just noticed the class name is plural for some reason.

Reply:
Changed it, but there's another thing I now noticed. This class is placed in Validators folder in the AdminApi and in the Support folder in ConsumerApi? Should they both be in the Support or in the Validators (there is only one validator btw) folder?

Reply:
Good catch. I leave it up to you which one to choose.